### DISCUSSION TITLE
Support postgres Uuid

### DISCUSSION THREAD

**Discussion by tqwewe** [SeaQL/sea-orm#67] [Q&A] [answered]
I have a column of the following type:
```sql
"id" uuid DEFAULT uuid_generate_v4()
```
And I cannot find a row by it's ID.
I tried both of the following:
```rust
find_by_id("a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11")
// SELECT "users"."name" FROM "users" WHERE "users"."id" = 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11' LIMIT 1
// Error: Query Error: error returned from database: operator does not exist: uuid = text

find_by_id(uuid::Uuid::from_str("a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11").unwrap())
// ^^^^^^ the trait `From<uuid::Uuid>` is not implemented for `sea_orm::Value`
```
Is there any way to currently filter a row by [UUID](https://docs.rs/uuid)?

**Comment by tyt2y3** [comment, marked as answer]
https://github.com/SeaQL/sea-orm/blob/2e9c19dc97f2d93c766ccc0411e48ff81565c78b/tests/crud/create_cake.rs#L30-L40
Can you checkout this example?

**Reply by tqwewe** [reply]
Thanks! I think I just simply forgot to include the `"with-uuid"` flag.
